### DISCUSSION TITLE
Is stride 4.3 going to be released with  some major features like Bepu Physics etc ? And when?

### DISCUSSION THREAD

**Discussion by IndieDev99** [stride3d/stride#2486] [General]
Stride 4.2 was released in March and after that there is no news about any progress. No new tutorials on the YouTube channel.

**Comment by IXLLEGACYIXL** [comment]
releases are just a summary of what has been developed, you are always on the "almost latest" state of github

bepu is currently reviewed and when merged you can pull from github and test it and at some point if its tested for a bit a mini release happens https://github.com/stride3d/stride/tags mini releases happen based on stuff that got merged so relatively frequently

sooo you should check out the releases since march and check what isnt released yet ( 2 weeks ago was last release )

**Comment by VaclavElias** [comment]
This might be a better link, regarding releases: https://github.com/stride3d/stride/releases

Tutorials are created independenty from the releases. We are considering some tutorials, once Bepu is released.
